libs/extensions/std/exec/src/archive-interpreter-executor.ts

```diff
@@ -83,7 +83,7 @@ export class ArchiveInterpreterExecutor
             mimeType,
             content,
           );
-          const addedFile = root.putFile(relPath, file);
+          const addedFile = root.putFile('/' + relPath, file);
```

Comment:
We need to at a root-indicator here, since we defined in `InMemoryFileSystem`, that all paths should start at root. JSZip return here the relative Path starting from the zipped folder, so adding the root-indicator manually should be fine. --> Now the gtfs pipelines work again.

Reply:
This probably also should use a constant like `InMemoryFileSystem.PATH_SEPARATOR`?

Reply:
Yeah, to support this, i have to add an getter in the inmemoryfilesystem.